src/app/components/context-menu/asset-context-menu.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import * as ContextMenu from '@radix-ui/react-context-menu';
```

Comment:
@fabric-8, the way we have our UI library set up is that we export pre-styled radix components via `@leather.io/ui`. We should do the same here.

First, creating a new context menu component in mono/packages/ui, adding styling, exporting, then consuming here. Let me know if you need any guidance doing this, though if you look at some of the other components wrapping with `forwardRef` it's pretty straightforward